ui/src/main/webapp/wizards/ldap/stages/network-settings.js

```diff
@@ -32,87 +43,167 @@ const testConnect = (conn) => ({
-      <Description>
-        To establish a connection to the remote LDAP store, we need the hostname of the
-        LDAP machine, the port number that the LDAP service is running on, and the
-        encryption method. Typically, port 636 uses LDAPS encryption and port 389 uses
-        StartTLS.
-      </Description>
-
-      <Body>
-        <Hostname
-          value={configs.hostname}
-          errorText={errors.hostname}
-          onEdit={onEdit('hostname')}
-          autoFocus />
-
-        <Port
-          value={configs.port}
-          errorText={isPortInvalid ? getFriendlyMessage('INVALID_PORT_RANGE') : errors.port}
-          onEdit={onEdit('port')}
-          options={[389, 636]} />
-
-        <Select
-          value={configs.encryption}
-          errorText={errors.encryption}
-          onEdit={onEdit('encryption')}
-          label='Encryption Method'
-          options={[ 'none', 'ldaps', 'startTls' ]} />
-
-        <Navigation>
-          <Back onClick={prev} />
-          <Next disabled={isPortInvalid}
+class NetworkSettings extends Component {
+  constructor (props) {
+    super(props)
+    this.state = { selected: [] }
+  }
+  filterUpdateHosts () {
+    const {
+        onEdit,
+        configs: {
+          connectionInfo = {}
+        } = {}
+      } = this.props
+
+    const filtered = Object.keys(connectionInfo).filter((_, i) => {
+      return this.state.selected.indexOf(i) === -1
+    }).reduce((o, key) => {
+      o[key] = connectionInfo[key]
+      return o
+    }, {})
+
+    onEdit('connectionInfo')(filtered)
+    this.setState({ selected: [] })
+  }
+
+  render () {
+    const {
+      client,
+      setDefaults,
+      prev,
+      next,
+      configs,
+      configs: {
+        connectionInfo = {}
+      } = {},
+      onEdit,
+      onError,
+      onStartSubmit,
+      onEndSubmit
+    } = this.props
+
+    const { messages, ...errors } = groupErrors([
+      'hostname',
+      'port',
+      'encryption'
+    ], this.props.errors)
+
+    const isPortInvalid = configs.port === undefined || configs.port < 0 || configs.port > 65535
+
+    const missingHost = configs.connectionInfo === undefined || Object.keys(configs.connectionInfo).length === 0
+
+    return (
+      <div>
+        <Mount
+          on={setDefaults}
+          port={636}
+          encryption='ldaps'
+          loadbalancing='round_robin' />
+
+        <Title>LDAP Network Settings</Title>
+        <Description>
+          To establish a connection to the remote LDAP store, we need the hostname of the
+          LDAP machine, the port number that the LDAP service is running on, and the
+          encryption method. Typically, port 636 uses LDAPS encryption and port 389 uses
+          StartTLS.  Adding more than one host creates a cluster.  The cluster connections can
+          be load balanced between the hosts (round-robin) or can be treated as a failover cluster
+          where the first host is considered the primary host.
```

Comment:
❓ Thoughts on leaving a note that the configurations of each of the LDAPS are assumed to be the same?